### DISCUSSION TITLE
VolumeSnapshot failed to create: nil Timestamp

### DISCUSSION THREAD

**Discussion by IamRFC1918** [longhorn/longhorn#3501] [Q&A] [answered]
## Question
Hi there,

im trying to create Volumesnapshots. When I create a VolumeSnapshot CR, like the example in the docs, or try to create a backup with kasten.io this error occurs in the longhorn csi-snapshotter pod:

E0105 09:36:28.492412       1 snapshot_controller.go:122] checkandUpdateContentStatus [snapcontent-1e3e8247-188d-4c44-ad60-8d7545642215]: error occurred failed to take snapshot of the volume, pvc-4446f00a-e1dd-484b-a3d0-4006aeefdb4a: "timestamp: nil Timestamp"
E0105 09:36:28.492573       1 snapshot_controller_base.go:261] could not sync content "snapcontent-1e3e8247-188d-4c44-ad60-8d7545642215": failed to take snapshot of the volume, pvc-4446f00a-e1dd-484b-a3d0-4006aeefdb4a: "timestamp: nil Timestamp"

Longhorn 1.2.3 on rke cluster. I've installed the snapshot controller with this manual: https://longhorn.io/docs/1.2.3/snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support/

Does someone know where the error is?

Thanks for your help!

> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.

## Environment

 - Longhorn version: 1.2.3
 - Kubernetes version: 1.21.7 (rke)
 - Node config
   - OS type and version flatcar linux, latest
   - CPU per node: 4
   - Memory per node: 4GB
   - Disk type HDD
   - Network bandwidth and latency between the nodes:  same kvm host
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): KVM

**Comment by jenting** [comment, marked as answer]
Same as https://github.com/longhorn/longhorn/issues/3392

**Comment by innobead** [comment]
This is actually an issue to improve. Thanks for reporting this, let's follow up with #3392 .
